Makefile

```diff
@@ -431,10 +450,18 @@ ifndef VERSION
 endif
 	git tag $(VERSION)
 
-	CALICOCTL_NODE_VERSION=$(VERSION) $(MAKE) dist/calicoctl calico/ctl calico/node
+	# Build the calicoctl binaries, as well as the calico/ctl and calico/node images.
+	CALICOCTL_NODE_VERSION=$(VERSION) $(MAKE) dist/calicoctl dist/calicoctl-darwin-amd64 dist/calicoctl-windows-amd64 
+	CALICOCTL_NODE_VERSION=$(VERSION) $(MAKE) calico/ctl calico/node
+
 	# Check that the version output includes the version specified.
-# Tests that the "git tag" makes it into the binary. Main point is to catch "-dirty" builds
+# Tests that the "git tag" makes it into the binaries. Main point is to catch "-dirty" builds
+# Release is currently supported on darwin / linux only.
+ifeq ($(OS),darwin)
+	if ! dist/calicoctl-darwin-amd64 version | grep 'Version:\s*$(VERSION)$$'; then echo "Reported version:" `dist/calicoctl-darwin-amd64 version` "\nExpected version: $(VERSION)"; false; else echo "Version check passed\n"; fi
```

Comment:
It seems that you now have to set OS=darwin when running the make target - I thought it would be marginally easier if we didn't have to do that.